test/export/InstanceExporter.test.ts

```diff
@@ -1998,6 +1998,51 @@ describe('InstanceExporter', () => {
       });
     });
 
+    it('should something', () => {
```

Comment:
Oops.  Forgot to describe the test.
```suggestion
    it('should handle extensions on non-zero element of primitive arrays', () => {
```